fix: auto scroll behavior (#31541)
cypress-io/cypress: cli/CHANGELOG.md

<!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
## 14.3.2

_Released 4/22/2025 (PENDING)_

**Bugfixes:**

> - Fixed and issue where auto scroll in the Cypress Reporter was not happening correctly. Fixes [#31530](https://github.com/cypress-io/cypress/issues/31530).

Reviewer: ```suggestion
- Fixed an issue where auto scroll in the Cypress Command Log was not scrolling correctly. Fixes [#31530](https://github.com/cypress-io/cypress/issues/31530).
```
